cmake/globalconfig.cmake

```diff
@@ -99,11 +99,11 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_LIST_DIR}/modules")
 
 #--------------------------------------------------------------------
 # Add parameter for paths to external modules
-set(IVW_EXTERNAL_MODULES "" CACHE STRING "Paths to directory containing external modules")
+set(IVW_EXTERNAL_MODULES "" CACHE STRING "Semicolon (;) separated paths to directories containing external modules")
 
 #--------------------------------------------------------------------
 # Add parameter for paths to external projects
-set(IVW_EXTERNAL_PROJECTS "" CACHE STRING "Paths to directory containing external projects")
+set(IVW_EXTERNAL_PROJECTS "" CACHE STRING "Semicolon (;) separated paths to directories containing cmake config file (.cmake/CMakeLists.txt file)")
```

Comment:
What do you mean by .cmake/?

Reply:
Should probably not call CMakeLists.txt files Cmake config file as that might be confused with FooConfig.cmake files...

Reply:
I'll adjust and say that add_directory will be called